### PR TITLE
Implement new layout

### PR DESCRIPTION
**Pull request by travigd**
This PR is a bit large, I'm sowwy!

In the future, I think I'd like to avoid having dropdown menus for the navigation and just implement a collapsible side-navigation (like what's present before this PR) for things that deserve it. I think that makes sense for those kinds of things because you get a nice "table of contents" type view.

Please let me know if you have any comments/questions/concerns about the layout or design or anything (very much a non-final WIP!).

### CODE REVIEW

**Comment by travigd** [comment]
Think it looks pretty decent! The hero on mobile is sub-optimal, but I don't think that's a huge deal for now.

localhost_3000_(Moto G4)
localhost_3000_(Moto G4) (1)
localhost_3000_

File: src/pages/about.mdx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+### Passion
+
+We are dedicated to the relentless pursuit of our vision.
+
+### Grit
+
+We push forward with courage and determination, regardless of the circumstances,
+to achieve our mission.
+
+Visit our oSTEM Code of Conduct
```
Comment (cmytelka) [inline src/pages/about.mdx:46]: This should be a link yeah?

File: src/pages/about.mdx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+(SHPE), National Society of Black Engineers (NSBE), Society of Asian Scientists
+and Engineers (SASE), American Indian Science and Engineering Society (AISES),
+and others.
+
+In order to advance LGBTQ+ STEM students, five chapters collaborated to found
+oSTEM Inc. in 2009 with 501(c)(3) status by December 2010. In 2017, oSTEM, Inc.
+expanded its mission to be inclusive of professionals, serving all LGBTQ+ people
+in the STEM community.
+
+## Privacy Policy
```
Comment (cmytelka) [inline src/pages/about.mdx:83]: Link to our privacy policy here (I think we have one right?)

File: src/components/layout/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+          >
+            <FontAwesomeIcon icon={faEnvelope} className="mr-2" />
+            info@ostem.org
+          </a>
+        </div>
+        <div className="flex flex-wrap justify-center items-center">
+          <SocialRow />
+        </div>
+      </div>
+      <div className="text-center">
```
Comment (cmytelka) [inline src/components/layout/Footer.tsx:28]: Should we link to Privacy Policy and Purchase Policy here like in oSTEM.org?

**Review by cmytelka — APPROVED** [review]
Didn't understand everything here but it looks good!

**Pull request merged**